### ISSUE TITLE
unable to uncheck deprecate

### ISSUE THREAD

**Issue by kbserm** [OAGi/Score#996]
Will have to try to reproduce this, but the guess is:

1. revise an ACC.
2. deprecate an association
3. move to candidate.
4. move back to revise again to try to undeprecate the association. The deprecate flag is locked.

**Comment by asdimitriadis** [comment]
I could not reproduce this issue in Score 2.0

**Comment by dubnemo** [comment]
@kbserm @asdimitriadis I don't remember the sequence I went through on the Architecture call, not exactly as Serm guesses. 
 
I do remember I went to WIP, tried to uncheck, and it was greyed out.
then I tried to Cancel and it was still there. 
When I clicked Revise again, then it was no longer there.

We questioned this was a bug or not, or a caching issue.  I stated it was 'odd' but certainly low priority.
